kernel/libc/newlib/newlib_exit.c

```diff
@@ -7,6 +7,8 @@
 
 #include <arch/arch.h>
 
+extern void post_newlib_exit(int ret_code) __noreturn;
+
 void _exit(int code) {
```

Comment:
I know you don't have to do it, but can we declare this extern? I'd like to start making things "extern" that are coming from newlib or are getting called from newlib.